base/cmd.jl

```diff
@@ -41,6 +41,7 @@ has_nondefault_cmd_flags(c::Cmd) =
 
 """
     Cmd(cmd::Cmd; ignorestatus, detach, windows_verbatim, windows_hide, env, dir)
+    Cmd(exec::Vector{String})
```

Comment:
Should be noted that this constructor does *not* copy `exec`, so any subsequent changes to `exec` will be reflected in the `Cmd` object.